### ISSUE TITLE
Project Name on file system should be uniq

### ISSUE THREAD

**Issue by AndreyKomarov** [SitecoreSupport/PatchCreator#41]
Could you please make sure that PatchCreator will create project names including Sitecore version?

Currently, if you need to create a few patches for the same issue and are using the same folder for storage, you have to rename or delete previous project folder:
http://www.screencast.com/t/N2kLHok3KKbY

**Comment by AlenPelin** [comment]
Valid point, noted. 

But frankly speaking I don't see any need in keeping projects after you submitted the patch - you can always get it either from GitHub or from our backup folder on build server.

**Comment by AlenPelin** [comment]
@RomanChernyk

**Comment by RomanChernyk** [comment]
Not sure I got it. Don't we just switch between branches within a single folder when developing fixes for different versions?

So, I would see the workflow as follows:
1. Create an initial branch for 7.2.0.
2. Complete the development.
3. Crete another branch for 8.2.0.
4. ...

**Comment by AlenPelin** [comment]
I suppose he wanted to use patch creator to create branch for 8.2.0, and in this case it will create new repo on file system.

**Comment by RomanChernyk** [comment]
Oh, then this looks like a bug.

**Comment by AlenPelin** [comment]
No, we didn't include this scenario to our list :) so it is just a missing feature. 

I propose either to create branch manually and update nugets, or delete old repository before using patch creator.

**Comment by RomanChernyk** [comment]
> I propose either to create branch manually and update nugets

I would not suggest this to engineers, I saw lot's of problems with this approach.

Better to delete the old repo.

> we didn't include this scenario to our list :) so it is just a missing feature.

Ok :)

**Comment by AlenPelin** [comment]
Could you update the readme with this "known issue"?
